src/main/java/dk/kb/cfutv/web/CfuTvServlet.java

```diff
@@ -341,9 +323,9 @@ public Response getStatus(@QueryParam("filename") String filename){
 			}
 			double size = ((double) f.length()) / 1024;
 			long datetime = f.lastModified();
-			Date date = new Date(datetime);
-			SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
-			String lastModified = simpleDateFormat.format(date);
+			ZonedDateTime date = ZonedDateTime.ofInstant(Instant.ofEpochMilli(datetime), ZoneId.of("Europe/Copenhagen"));
+			DateTimeFormatter format = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss", Locale.ROOT).withZone(ZoneId.of("Europe/Copenhagen"));
```

Comment:
Ville oprette static final String format  = "yyyy-MM-dd HH:mm:ss" i stedet for. Den bruges så mange gange og vil være besværlig at rette til en anden pattern, hvis det er. Det vil også gøre koden lidt bedre læsbare.